test/fixtures/export/Do not caps-protect name fields #384.biblatex

```diff
@@ -27,11 +28,13 @@
 }
 
 @book{bovendeert,
-  author = {{boven d'Eert}, Christianus}
+  author = {boven d' Eert, Christianus},
+  options = {useprefix=true}
 }
 
 @book{s-gravesande,
-  author = {'s-Gravesande, Goverdus}
+  author = {'s- Gravesande, Goverdus},
```

Comment:
OK, tests are running on that at https://travis-ci.org/ZotPlus/zotero-better-bibtex/jobs/88983180